### PR TITLE
Add content/pt-br/docs/reference/glossary/rbac.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translate content/pt-br/docs/reference/glossary/rbac.md

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/rbac.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: RBAC (Controle de Acesso Baseado em Funções)
+id: rbac
+date: 2018-04-12
+full_link: /docs/reference/access-authn-authz/rbac/
+short_description: >
+  Gerencia decisões de autorização, permitindo que os administradores configurem dinamicamente políticas de acesso por meio da API do Kubernetes.
+
+aka:
```
Comment (sftim) [inline content/pt-br/docs/reference/glossary/rbac.md:9]:
```suggestion
aka:
 - Role Based Access Control
 - Controle de Acesso Baseado em Funções
```
This is where I'd put the alternative name (and in the title, I'd leave it just as RBAC). What do you think?

**Review by sftim — COMMENTED** [review]
Somewhere in this content I'd expect to explain that RBAC expands in English to Role Based Access Control.

File: content/pt-br/docs/reference/glossary/rbac.md (hunk 1 above)
Comment (MrErlison) [reply content/pt-br/docs/reference/glossary/rbac.md:9]: Nice. Thank you.

**Comment by rikatz** [comment]
/lgtm
/approve
Thanks

**Pull request merged**